components/brave_wallet_ui/components/desktop/views/portfolio/index.tsx

```diff
@@ -215,6 +225,26 @@ const Portfolio = (props: Props) => {
     }
   }, [portfolioHistory, portfolioBalance])
 
+  const selectedAssetTransactions = React.useMemo((): TransactionInfo[] => {
+    const list = transactions.map((account) => {
+      return account?.transactions
+    })
+    const combinedList = [].concat.apply([], list)
+    if (selectedAsset?.symbol === selectedNetwork.symbol) {
+      return combinedList.filter((tx: TransactionInfo) => tx.txType === TransactionType.ETHSend || tx.txType === TransactionType.ERC20Approve)
+    } else {
+      return combinedList.filter((tx: TransactionInfo) => tx.txData.baseData.to.toLowerCase() === selectedAsset?.contractAddress.toLowerCase())
+    }
+  }, [selectedAsset, transactions])
+
+  const findAccount = (address: string): WalletAccountType | undefined => {
+    return accounts.find((account) => address.toLowerCase() === account.address.toLowerCase())
+  }
```

Comment:
Could this be memoized?

Reply:
Resolved through slack discussion, is not need here! 😎